templates/tab/js/default/public/auth-start.html

```diff
@@ -23,30 +24,29 @@
         localStorage.removeItem("codeVerifier");
         var currentURL = new URL(window.location);
         var clientId = currentURL.searchParams.get("clientId");
-
+        var tenantId = currentURL.searchParams.get("tenantId");
```

Comment:
can we use `context.tid`? Since you use tenant id value from Teams context in auth-end.html, we're better to align the source of this value.

Reply:
This tenantId is not used, thus delete this line. context.tid is used below.